src/main/java/org/sunbird/workflow/controller/ProfileWorkflowController.java

```diff
@@ -0,0 +1,25 @@
+package org.sunbird.workflow.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.sunbird.workflow.models.Response;
+import org.sunbird.workflow.models.SearchCriteria;
+import org.sunbird.workflow.service.Workflowservice;
+
+@RestController
+@RequestMapping("/v2/workflow")
+public class ProfileWorkflowController {
```

Comment:
v1/workflow -- path was in WorkflowController
so,
v2/workflow -- must be in WorkflowControllerV2 -- not in ProfileWorkflowController class

Reply:
This is done.